haystack/utils.py

```diff
@@ -8,6 +8,7 @@
 from haystack.document_store.sql import DocumentORM
```

Comment:
@tholor @oryx1729 What do you suggest about moving `convert_labels_to_squad` to separate file?
I worry about circular dependency due to `DocumentORM` import. I faced similar issue in Obsei and it was hard to resolve so better to avoid beginning.